flocker/node/agents/ebs.py

```diff
@@ -590,6 +602,11 @@ def destroy_volume(self, blockdevice_id):
                         'Failed to delete volume: {!r}'.format(blockdevice_id)
                     )
                 else:
+                    ebs_volume = self._get_ebs_volume(blockdevice_id)
```

Comment:
Thanks for pointing out! Calling ``_get_ebs_volume`` before ``destroy_volume`` is definitely simpler. Fixed.